duckduckgo/Android
# Add privacy terms

Asana Issue URL: https://app.asana.com/0/488551667048375/471193197990041

## Description
Adds terms of service to privacy dashboard.

There is an outstanding question around the algorithm in https://app.asana.com/0/261348710882778/495254191196650. If anything changes, it will be captured in a new task.

## Steps to Test this PR:
TEST ONE:
1. Visit duckduckgo.com
1. Ensure terms are "good"

TEST TWO (poor by classification):
1. Visit google.com
1. Ensure practices are "poor"

TEST THREE (poor by score):
1. Visit amazon.com
1. Ensure practices are "poor"

TEST FOUR
1. Visit netflix.com
1. Ensure practices are "mixed"

TEST FIVE
1. Visit subsymbnolic.net
1. Ensure practices are "unknown"
